### DISCUSSION TITLE
first: last: not working

### DISCUSSION THREAD

**Discussion by J-Rigondo** [tailwindlabs/tailwindcss#8556] [Help]
hello!
I try to do according to doc, https://tailwindcss.com/docs/hover-focus-and-other-states
but
even, odd, empty keyword is working but first, last keyword is not working.
how can I do this?

className="flex justify-between my-2 first:bg-blue-300 even:bg-blue-300"

**Comment by crswll** [comment]
Hard to tell what you might be doing wrong without code to look at. My guess is you're trying to put the class on a common parent  insteead of each sibling. This example should help: https://play.tailwindcss.com/H2tLSiaQ7x
